### DISCUSSION TITLE
How to disable multi-screen snapshots

### DISCUSSION THREAD

**Discussion by Oddimus** [pubpub-zz/ppInk#327] [Q&A]
Kindly assist, how do I disable muilt-screen snapshot...

**Comment by pubpub-zz** [comment]
1st option: select the area you are looking for
2nd option: set ppInk in window mode

**Comment by Oddimus** [comment]
thanks.

**Reply by pubpub-zz** [reply]
do not hesitate to star the project

**Comment by Oddimus** [comment]
Thanks, I will.
